U/Uno/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Uno"
+
+version = v"1.0.0"
+
+sources = [
+    GitSource(
+        "https://github.com/cvanaret/Uno.git",
+        "888114ce7d5e1d208ed8865dc836e4b94017e6a8",
+    ),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/Uno
+mkdir -p build
+cd build
+
+cmake \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    ..
+
+make -j${nproc}
+make install
+"""
+
+platforms = supported_platforms()
+
+products = [
+    ExecutableProduct("Uno", :amplexe),
```

Comment:
IMO, it would be better to name this similar to the actual executable name instead of calling it `amplexe`.

Reply:
We've used `amplexe` as the convention for all the other AMPL-compatible solvers:
https://github.com/jump-dev/AmplNLWriter.jl/tree/master?tab=readme-ov-file#precompiled-binaries

Reply:
Then I would suggest adding this reason as a comment in this file (on this line or next to it).